### PR TITLE
Remove 'api' from urls and other small changes

### LINKED ISSUE #74: The URLs all begin with `/api`. That is no longer needed.
Currently in stable we no longer require all api endpoints to be prefaced with `/api`. That will soon be released to production, and when it is, we should update the documentation to remove `/api`.

### LINKED ISSUE #77: Correct environment vars
The paw document has `stable` retailer/supplier envs that point to `stable.projectthanos.com`.

It should instead have `sandbox` retailer/supplier that point to `api-sandbox.cruxconnect.com`

### PR DESCRIPTION
**Pull request by baileymatthewr**
Remove 'api' from urls, use sandbox url as the default for the examples, correct the metadata for the documentation, and more.

Fixes #77
Fixes #74

### CODE REVIEW

File: _product_retailer/get_item_detail.md [hunk 1]
```diff
@@ -36,13 +36,13 @@ right_code: |
         "product_images": [
           {
             "uuid": "800388e7-de33-494a-94ac-93f57f959de3",
-            "url": "https://api.adorable.io/avatars/80/obad15.png",
+            "url": "https:/.adorable.io/avatars/80/obad15.png",
```
Comment (Gidgidonihah) [inline _product_retailer/get_item_detail.md]: Looks like an overzealous search and replace got rid of the correct adorable urls.

File: _product_retailer/get_items.md [hunk 2]
```diff
@@ -40,13 +40,13 @@ right_code: |
           "product_images": [
             {
               "uuid": "81ada0e1-74ee-4a32-b709-745950288608",
-              "url": "https://api.adorable.io/avatars/285/obad15.png",
+              "url": "https:/.adorable.io/avatars/285/obad15.png",
```
Comment (Gidgidonihah) [inline _product_retailer/get_items.md]: More borked adorable urls

File: _product_retailer/get_sku_detail.md [hunk 3]
```diff
@@ -41,13 +41,13 @@ right_code: |
     "product_images": [
       {
         "uuid": "1929dd2b-ad2c-4021-9960-fb26e7bf448a",
-        "url": "https://api.adorable.io/avatars/80/obad20.png",
```
Comment (Gidgidonihah) [inline _product_retailer/get_sku_detail.md]: adoraborked.

File: _product_retailer/get_skus.md [hunk 4]
```diff
@@ -60,13 +60,13 @@ right_code: |
       "product_images": [
         {
           "uuid": "1929dd2b-ad2c-4021-9960-fb26e7bf448a",
-          "url": "https://api.adorable.io/avatars/80/obad20.png",
+          "url": "https:/.adorable.io/avatars/80/obad20.png",
```
Comment (Gidgidonihah) [inline _product_retailer/get_skus.md]: adoraborked.

File: _product_retailer/search_items.md [hunk 5]
```diff
@@ -289,13 +289,13 @@ right_code: |
         "product_images": [
           {
             "uuid": "443ba02e-973b-4fee-8a00-9c4bd8c5c3a5",
-            "url": "https://api.adorable.io/avatars/80/obad21.png",
+            "url": "https:/.adorable.io/avatars/80/obad21.png",
```
Comment (Gidgidonihah) [inline _product_retailer/search_items.md]: adoraborked.

**Review by Gidgidonihah — CHANGES_REQUESTED** [review]
The only thing I saw that's an issue are the borked adorable urls. They may be more that I didn't catch.

**Comment by Gidgidonihah** [comment]
Looks like it also includes some work on #73

**Comment by baileymatthewr** [comment]
@Gidgidonihah : I made the changes you recommended

**Review by Gidgidonihah — APPROVED** [review]

**Review by jtprince — APPROVED** [review]

**Pull request merged**